packages/web-components/src/presence-badge/presence-badge.styles.ts

```diff
@@ -0,0 +1,107 @@
+import { css } from '@microsoft/fast-element';
+
+export const styles = css`
+  :host {
+    align-items: center;
+    box-sizing: border-box;
+    cursor: default;
+    display: inline-flex;
+    flex-direction: row;
+    font-size: 0;
+    line-height: 0;
+    min-height: 6px;
+    min-width: 6px;
+    position: relative;
+    transition: opacity 0.5s linear;
+    --presence-size: 16px;
+    --presence-mask-width: 2px;
```

Comment:
Curious if we think there's enough need to have these be something other than pixels? Would this potentially be more ergonomic as a number which is converted to pixels inline? What are the potential implications either way?

Reply:
@chrisdholt - Based on the react presence badge spec, would we consider going a different direction altogether: adding a size attribute to match the fluent sizes (smallest, smaller, small, medium, large, larger, largest)?

Reply:
It's _always_ pixels in Fluent, but someone may want to override that in their own system. I like allowing that flexibility, but maybe it's better to hew closely to units specified in the Fluent spec